ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -1,5 +1,74 @@
 [
   {
+    "id": "java-centos-mysql",
```

Comment:
@davidfestal @sunix it makes sense to me to merge this stack to eclipse/che master